packages/hub/writers.js

```diff
@@ -132,7 +123,22 @@ class Writers {
     }
     await this.pgSearchClient.ensureDatabaseSetup();
 
-    let schema = await this.currentSchema.getSchema();
+    let addedModels = [];
```

Comment:
I think we can layer this better so it's clear what's the difference between the new outer layer and the original layer. Also, I think the layering can look the same for each of `create()`, `update()`, and `delete()`.

What I mean is, each of those methods should have only two lines dedicated to special behavior for the card type:

```js
if (type == 'cards') {
  return this.handleCardOperations(...);
}
```

And then handleCardOperations is responsible for taking the card and figuring out a list of new calls to `create`, `update`, and/or `delete` to make, and running them.

Reply:
cool--will do.